specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2319,6 +2322,21 @@
leIdentityCardNumber", "ChinaResidentIdentityCardPRCNumber", "HongKongIdentityCardHKIDNumber", "IndiaUniqueIdentificationAadhaarNumber", "IndiaPermanentAccountNumberPAN", "IndonesiaIdentityCardKTPNumber", "PhilippinesUnifiedMultiPurposeIDNumber", "BrazilCPFNumber", "BrazilNationalIDCardRG", "BrazilLegalEntityNumberCNPJ", "DenmarkPersonalIdentificationNumber", "BelgiumNationalNumber", "GermanyIdentityCardNumber", "IrelandPersonalPublicServicePPSNumber", "NetherlandsCitizenServiceBSNNumber", "NorwayIdentityNumber", "PortugalCitizenCardNumber", "CroatiaPersonalIdentificationOIBNumber", "CroatiaIdentityCardNumber", "CzechPersonalIdentityNumber", "GreeceNationalIDCard", "SouthAfricaIdentificationNumber", "InternationalClassificationofDiseasesICD10CM", "InternationalClassificationofDiseasesICD9CM", "AzurePublishSettingPassword", "AzureStorageAccountKeyGeneric", "SQLServerConnectionString", "AzureDocumentDBAuthKey", "AzureIAASDatabaseConnectionStringandAzureSQLConnectionString", "AzureIoTConnectionString", "AzureRedisCacheConnectionString", "AzureSAS", "AzureServiceBusConnectionString", "AzureStorageAccountKey", "EUPhoneNumber", "EUMobilePhoneNumber", "EUPassportNumber", "EUNationalIdentificationNumber", "EUSocialSecurityNumberSSNorEquivalentID", "EUDriverLicenseNumber", "EUTaxIdentificationNumberTIN", "EUGPSCoordinates", "AustralianCompanyNumber", "AustriaVATNumber", "UkrainePassportNumberInternational", "UkrainePassportNumberDomestic", "FinlandEuropeanHealthInsuranceNumber", "FranceHealthInsuranceNumber", "RussianPassportNumberDomestic", "RussianPassportNumberInternational", "NewZealandBankAccountNumber", "NewZealandInlandRevenueNumber", "AustralianBusinessNumber", "PolishREGONNumber", "SwissSocialSecurityNumberAHV", "GermanyValueAddedTaxNumber", "JapaneseMyNumberPersonal", "JapaneseMyNumberCorporate", "NetherlandsValueAddedTaxNumber", "ItalyValueAddedTaxNumber", "HungarianValueAddedTaxNumber", "FranceValueAddedTaxNumber", "BelgiumValueAddedTaxNumber", "NewZealandDriverLicenseNumber", "NewZealandSocialWelfareNumber", "AustriaTaxIdentificationNumber", "LuxemburgNationalIdentificationNumberNaturalpersons", "NetherlandsTaxIdentificationNumber", "SloveniaTaxIdentificationNumber", "LuxemburgNationalIdentificationNumberNonnaturalpersons", "PolandTaxIdentificationNumber", "BulgariaUniformCivilNumber", "AustriaIdentityCard", "CyprusTaxIdentificationNumber", "CyprusIdentityCard", "EstoniaPersonalIdentificationCode", "GreekTaxidentificationNumber", "HungaryPersonalIdentificationNumber", "HungaryTaxidentificationNumber", "ItalyFiscalCode", "LithuaniaPersonalCode", "MaltaIdentityCardNumber", "SpainTaxIdentificationNumber", "UKUniqueTaxpayerReferenceNumber", "FranceTaxIdentificationNumber", "GermanyTaxIdentificationNumber", "PortugalTaxIdentificationNumber", "RomaniaPersonalNumericalCode", "SlovakiaPersonalNumber", "CroatiaNationalIDCardNumber", "SloveniaUniqueMasterCitizenNumber", "SwedenTaxIdentificationNumber", "LatviaPersonalCode", "SpainDN", "all", "default"]
```

Comment:
**A**ll, **D**efault

Reply:
The problem with an enum is that any new types will require an API version change vs. a model version change. 
We should ask the API board and SDK team if we can have a string instead and simply document the permitted values.

Reply:
this is now an extensible enum, so we should be good here - also noted that we don't intend to add types very often (6 month cadence or so is what a PM briefly mentioned)